src/main/python/deep_qa/training/trainer.py

```diff
@@ -518,6 +573,7 @@ def _save_best_model(self):
         epoch_weight_file = "%s_weights_epoch=%d.h5" % (self.model_prefix, self.best_epoch)
         final_weight_file = "%s_weights.h5" % self.model_prefix
         copyfile(epoch_weight_file, final_weight_file)
+        print("Saved the best model to {}".format(final_weight_file))
```

Comment:
You need [these lines](https://github.com/allenai/deep_qa/blob/d4157d35d69dd7df5beece83bec58f64aca5a0f3/src/test/python/deep_qa_test/models/reading_comprehension/bidirectional_attention_test.py#L16-L18), which should go into a `DeepQaTestCase` super class at some point.